CHANGELOG.latest.md

```diff
@@ -1,11 +1,49 @@
-- Eighth beta for RevenueCat framework 🎉
+- Ninth beta for RevenueCat framework 🎉
     100% Swift framework + ObjC support.
+
+[Full Changelog](https://github.com/revenuecat/purchases-ios/compare/4.0.0-beta.8...HEAD)
```

Comment:
Should this compare only the 2 releases?

Reply:
Good point, I think comparing with just the previous release makes sense.

Reply:
shoot I just saw this comment after merging, I'll update the release notes in the GH release, though. Good catch, I made the link before the branch was created so I couldn't compare